base/setup/reactos/drivepage.c

```diff
@@ -562,16 +940,71 @@ GetPartitionTypeString(
     }
 }
 
+
+static VOID
+PrettifySize1(
+    _Inout_ PULONGLONG Size,
+    _Out_ PCWSTR* Unit)
+{
+    ULONGLONG DiskSize = *Size;
+
+    if (DiskSize >= 10 * GB) /* 10 GB */
+    {
+        DiskSize = RoundingDivide(DiskSize, GB);
+        *Unit = L"GB"; // MUIGetString(STRING_GB);
+    }
+    else
+    {
+        DiskSize = RoundingDivide(DiskSize, MB);
+        if (DiskSize == 0)
+            DiskSize = 1;
+        *Unit = L"MB"; // MUIGetString(STRING_MB);
+    }
+
+    *Size = DiskSize;
+}
+
+static VOID
+PrettifySize2(
+    _Inout_ PULONGLONG Size,
+    _Out_ PCWSTR* Unit)
+{
+    ULONGLONG PartSize = *Size;
+
+#if 0
```

Comment:
Why is this skipped?

Reply:
I copy-pasted the existing code from usetup. I think this was commented because we wanted to keep the partition size always in megabytes, because it is also in this unit that sizes for new partitions are being specified.